tests/e2e/standalone/standalone_test.go

```diff
@@ -58,6 +58,63 @@ func TestStandaloneInstall(t *testing.T) {
 	}
 }
 
+func TestNegativeScenarios(t *testing.T) {
+	// Ensure a clean environment
+	uninstall()
+	daprPath := getDaprPath()
+
+	homeDir, err := os.UserHomeDir()
+	require.NoError(t, err, "expected no error on querying for os home dir")
+
+	t.Run("run without install", func(t *testing.T) {
+		output, err := spawn.Command(daprPath, "run", "test")
+		require.NoError(t, err, "expected no error status on run without install")
+		path := filepath.Join(homeDir, ".dapr", "components")
+		require.Contains(t, output, path+": no such file or directory", "expected output to contain message")
+	})
+
+	t.Run("list without install", func(t *testing.T) {
+		output, err := spawn.Command(daprPath, "list")
+		require.NoError(t, err, "expected no error status on list without install")
+		require.Equal(t, "No Dapr instances found.\n", output)
+	})
+
+	t.Run("stop without install", func(t *testing.T) {
+		output, err := spawn.Command(daprPath, "stop", "-a", "test")
+		require.Error(t, err, "expected error status on list without install")
+		require.Contains(t, output, "failed to stop app id test: couldn't find app id test", "expected output to match")
+	})
+
+	t.Run("stop unkonwn flag", func(t *testing.T) {
+		output, err := spawn.Command(daprPath, "stop", "-p", "test")
+		require.Error(t, err, "expected error on stop with unknown flag")
+		require.Contains(t, output, "Error: unknown shorthand flag: 'p' in -p\nUsage:", "expected usage to be printed")
+		require.Contains(t, output, "-a, --app-id string   The application id to be stopped", "expected usage to be printed")
+	})
+
+	t.Run("run unknown flags", func(t *testing.T) {
+		output, err := spawn.Command(daprPath, "run", "--flag")
+		require.Error(t, err, "expected error on run unkonwn flag")
+		require.Contains(t, output, "Error: unknown flag: --flag\nUsage:", "expected usage to be printed")
+		require.Contains(t, output, "-a, --app-id string", "expected usage to be printed")
+		require.Contains(t, output, "The id for your application, used for service discovery", "expected usage to be printed")
+	})
+
+	t.Run("uninstall without install", func(t *testing.T) {
+		output, err := spawn.Command(daprPath, "uninstall", "--all")
+		require.NoError(t, err, "expected no error on uninstall without install")
+		require.Contains(t, output, "Removing Dapr from your machine...", "expected output to contain message")
+		path := filepath.Join(homeDir, ".dapr", "bin")
+		require.Contains(t, output, "WARNING: "+path+" does not exist", "expected output to contain message")
+		require.Contains(t, output, "WARNING: dapr_placement container does not exist", "expected output to contain message")
```

Comment:
These tests will fail when log message changes. Ideally, those could be a shared constant.

Reply:
I mean... changing them later is not hard.